docs/core/versions/index.md

```diff
@@ -19,9 +19,13 @@ This article aims at clarifying how the .NET Core SDK and runtime are versioned.
 
 There are lots of moving parts that version independently in .NET Core. However, starting with .NET Core 2.0, there is an easy to understand top-level version number that everybody understands to be *the* version of ".NET Core" as a whole. The rest of this document goes into the details of the versioning of all those parts. These details can be important if you're a package manager, for example.
 
+> [!IMPORTANT]
+> The versioning details explained on this topic don't apply to the current version of the .NET Core SDK and runtime.
+> The version scheme is changing on future releases and you can see the current proposal at the [dotnet/designs](https://github.com/dotnet/designs/pull/29) repository.
```

Comment:
on future releases --> in future releases
Instead of "and you can see", it's better to have a new sentence ("You can see the current...")